client/src/components/RatingsAndReviews/RatingsAndReviews.jsx

```diff
@@ -34,7 +34,7 @@ const RatingsAndReviews = function({ onChangeFileHandler, onClickUploadHandler,
     <div id="ratings-main" data-testid="ratings-main">
       <div id="ratings-left-pane">
         <div id="left-pane-title">
-          <h2>RatingsAndReviews!</h2>
+          RatingsAndReviews!
```

Comment:
you didn't have to delete the h2 hahaha, I just meant use more specificity in your css to get only your h2 or whatever

Reply:
I got what I needed via className. We'll have a css talk this week and get some top-level style stuff set. I was just trying in vain to match the size and color of the wireframe spec.